README.md

```diff
@@ -6,9 +6,24 @@ This tap:
 
 - Pulls raw data from FrontApp's [API](https://dev.frontapp.com/)
 - Extracts the following resources from FrontApp
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
   - [Analytics](https://dev.frontapp.com/#analytics)
+=======
+  - [Analytics](https://dev.frontapp.com#analytics)
+>>>>>>> cleaned up http.py and added handling for when no data returned.
+=======
+  - [Analytics](https://dev.frontapp.com#analytics)
+>>>>>>> cleaned up http.py and added handling for when no data returned.
+=======
+  - [Analytics](https://dev.frontapp.com#analytics)
+>>>>>>> 1f95e607f623bdeea55b53cc16f1f3f007dda690
```

Comment:
Please remove/resolve these merge conflict markers. It looks like the unchanged line should just be kept and the added lines removed?